SCons/Script/SConscript.xml

```diff
@@ -408,23 +404,26 @@ in each of the specified directories.
 You may specify a name other than
 &SConscript;
 by supplying an optional
-<varname>name</varname>=<replaceable>script</replaceable>
-keyword argument.
+<varname>name</varname> keyword argument.
```

Comment:
I think better to leave the original name=script here as then users don't need to know what a keyword argument is..

Reply:
Restored, although I think "that ship has sailed" in terms of expecting to know what a kwarg is.